UIKitVK/UIKitVK/Model/Photo.swift

```diff
@@ -0,0 +1,31 @@
+// Photo.swift
+// Copyright © RoadMap. All rights reserved.
+
+import Foundation
+import RealmSwift
+
+// Ответ запроса
+struct ResultPhoto: Decodable {
+    let response: ResponsePhoto
+}
+
+// Список фото
+struct ResponsePhoto: Codable {
+    let count: Int
+    let photos: [Photo]
+
+    private enum CodingKeys: String, CodingKey {
+        case count
+        case photos = "items"
+    }
+}
+
+// Фото
+final class Photo: Codable {
+    let sizes: [SizePhoto]
+}
+
+// Ссылка на фото
```

Comment:
```suggestion
// Ссылка на фото
```
а назвал класс сайт это размер